docs/robots_file.md

```diff
@@ -0,0 +1,376 @@
+But validation will fail if you use a mismatched pair. e.g. bed 7 with 14 here.
+
+```code
+custom_robot(
+  'star-96-post-cap-pcr-purification',
+  name: 'STAR-96 LB Cap Lib PCR => LB Cap Lib PCR-XP',
+  verify_robot: false,
+  beds: {
+    bed(7).barcode => {
+      purpose: 'LB Cap Lib PCR',
+      states: ['passed'],
+      label: 'Bed 7'
+    },
+    bed(9).barcode => {
+      purpose: 'LB Cap Lib PCR-XP',
+      states: ['pending'],
+      label: 'Bed 9',
+      parent: bed(7).barcode,
+      target_state: 'passed'
+    },
+    bed(12).barcode => {
+      purpose: 'LB Cap Lib PCR',
+      states: ['passed'],
+      label: 'Bed 12'
+    },
+    bed(14).barcode => {
+      purpose: 'LB Cap Lib PCR-XP',
+      states: ['pending'],
+      label: 'Bed 14',
+      parent: bed(12).barcode,
+      target_state: 'passed'
+    }
+  }
+)
+```
+
+## A single labware preparation bed verification
+Sometimes you just want a bed verification to check that a single plate is on the liquid handler deck at the correct location, without a transfer happening. In these cases the lab is typically doing some sort of preparation process or chemistry on the plate.
+
+For these, you do not specify parent or child beds.
+
+For example, this bed verification checks a single plate is in bed 5 and changes it's state, but there is no parent or child definition:
+
+```code
+custom_robot(
+  'bravo-pf-post-shear-xp-prep',
+  name: 'Bravo PF Post Shear XP Preparation',
+  beds: {
+    bed(5).barcode => {
+      purpose: 'PF Post Shear XP',
+      states: ['started'],
+      label: 'Bed 5',
+      target_state: 'passed'
+    }
+  }
+)
+```
+
+NB. You do not have to change the state of a labware in a bed verification. It just makes sense in most cases to do so in order to control the appearance of buttons on the view in Limber and help 'lead' the user through their process.
+
+## A sequential series of transfers within the same bed verification
+Sometimes a liquid handler method performs a series of transfers in the same run. So rather than just A->B, it may perform A->B->C.
+
+See {PermissivePresenter} to create child plates while the parent plate is still in a pending state.
+
+Example with 3 steps:
+```code
+custom_robot(
+  'bravo-pf-post-shear-xp-to-pf-lib-xp',
+  name: 'Bravo PF Post Shear XP to PF Lib XP',
+  beds: {
+    car('1,3').barcode => {
+      purpose: 'PF Post Shear XP',
+      states: ['passed'],
+      label: 'Carousel 1,3'
+    },
+    bed(6).barcode => {
+      purpose: 'PF Lib',
+      states: ['pending'],
+      label: 'Bed 6',
+      target_state: 'passed',
+      parent: car('1,3').barcode
+    },
+    car('4,3').barcode => {
+      purpose: 'PF Lib XP',
+      states: ['pending'],
+      label: 'Carousel 4,3',
+      target_state: 'passed',
+      parent: bed(6).barcode
+    }
+  }
+)
+```
+
+Note above how the last two beds expect the plate to be in 'pending' state.
+And that they each have a parent of the previous bed.
```

Comment:
Ye this bit: “and that they each have a parent of the previous bed” could be added to the “parents” config explanation above.